openslides_backend/action/actions/motion/reset_state.py

```diff
@@ -67,12 +68,6 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         )
         timestamp = round(time.time())
         instance["last_modified"] = timestamp
-        if not motion.get("created"):
-            state = self.datastore.get(
-                fqid_from_collection_and_id("motion_state", instance["state_id"]),
-                ["set_created_timestamp"],
-                lock_result=False,
-            )
-            if state.get("set_created_timestamp"):
-                instance["created"] = timestamp
+        if not motion.get("workflow_timestamp"):
```

Comment:
reset_state resets the workflow_timestamp always, but set it again if  the state demands it:
```

instance["workflow_timestamp"] = None
set_workflow_timestamp_helper(self.datastore, instance, timestamp)
```

Reply:
Changed. in motion.reset_state.